oauth.go

```diff
@@ -0,0 +1,41 @@
+package pluginapi
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+// UserService exposes methods to manipulate OAuth Apps.
+type OAuthService struct {
+	api plugin.API
+}
+
+// Create creates a new OAuth App,
+//
+// Minimum server version: 5.38
+func (o *OAuthService) Create(app *model.OAuthApp) error {
+	createdApp, appErr := o.api.CreateOAuthApp(app)
+	if appErr != nil {
+		return normalizeAppErr(appErr)
+	}
+
+	*app = *createdApp
+
+	return nil
+}
+
+// Get gets an existing OAuth App.
```

Comment:
0/5 to match the comment for the `Delete` method like this

```suggestion
// Get gets an existing OAuth App by id.
```